<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextAnnotationEntryWidget</class>
 <widget class="QWidget" name="TextAnnotationEntryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>105</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Annotation Entry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l1">
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Edit Text Annotation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pText">
     <property name="toolTip">
      <string>Text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
